### DISCUSSION TITLE
Strimzi cert got auto renewed one month prior to the expiry date

### DISCUSSION THREAD

**Discussion by Jayaarthi** [strimzi/strimzi-kafka-operator#6225] [Q&A]
The CA cert (goes via key – “ca-cert”). this was suppose to expire on 16 Feb 2022. However, we saw that Stremzi renewed it on 17 Jan. why did it happen? Is there any setting that controls the renewal policy?

![image](https://user-images.githubusercontent.com/88392049/149954912-dd2aeaf6-4be2-455b-8121-f95587dad95f.png)

How to make changes in this policy configuration?

**Comment by scholzj** [comment]
I'm sorry => you clearly found the docs which describes why did it happened as well as how to change it. So, what is it actually you want to ask?

**Reply by Jayaarthi** [reply]
Thank you for the quick response,

Yes, you are right. I am trying to know the config exist in which secret because i have gone through related yamls and couldn't find where the configurations are made.

**Reply by scholzj** [reply]
You have to configure it in the `Kafka` custom resource. Something like `Kafka.spec.clusterCa.renewalDays` means it is in the `Kafka` custom resource, in the `spec` section, `clusterCa` subsection. This path might not exist in your resource => so you have to edit it and create the sections.
